### ISSUE TITLE
Embed into a Jupyter Notebook

### ISSUE THREAD

**Issue by kalugny** [QuantStack/jupyterlab-drawio#22]
Hi!
Thanks for a great plugin.

I have a request - 
I know it's possible to export and to embed the PNG, but it'd be great to embed the diagram itself which may be editable inside the notebook or that will open the editor on double-click.

Thanks!

**Comment by cutephoton** [comment]
Yes, it would be nice, wouldn't it? I would take this a step further. The killer feature would be deeper integration. The diagram should be parametrized or linked to workbook data in some form.

**Comment by nudurumatisk** [comment]
Definitely a killer feature, much like what is possible with Mathematica. Why not leverage Geogebra (and its Python View API)?

> Yes, it would be nice, wouldn't it? I would take this a step further. The killer feature would be deeper integration. The diagram should be parametrized or linked to workbook data in some form.

Definitely a killer feature, much like what is possible with Mathematica. Why not leverage Geogebra (and its Python View API if it's still available)?

**Comment by cutephoton** [comment]
Don't know about Geogebra. Seems like a different use case? I'm interesting in being able to draw conceptual electrical circuits and having part values updated once I feed parameters in to a series of calculations.

**Comment by gnzlbg** [comment]
Ideally this would work in such a way that math written in latex in the diagram gets rendered appropriately in the notebook (e.g. same size as the text independently of scaling).

**Comment by ragazzojp** [comment]
Instead of embedding the editor in a notebook, it's probably better to streamline the "export as png/svg" workflow, so that the diagrams are automatically exported when modified and updated inside notebooks using them.

**Comment by warmfrog** [comment]
Maybe this feature should open in jupyter-lab code base?
